tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -676,7 +676,7 @@ $KeepTopLevelVar: val;
   @KeepLessVar: val;
   @extend %KeepPlaceholderSelector;
 
-  #{$Keep}[#{$Keep}][#{Keep}^="#{Keep-1A}"]:#{$Keep} {
+  #{$Keep}[#{Keep}][#{Keep}^="#{Keep-1A}"]:#{$Keep} {
```

Comment:
this might not right ,  https://github.com/postcss/postcss-selector-parser/issues/211

Reply:
It is serious problem, a lot of projects uses interpolate for attribute selectors

Reply:
See https://github.com/postcss/postcss-selector-parser/issues/211